operation/fhir-operation-validate/src/main/java/com/ibm/fhir/operation/validate/type/ModeType.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.fhir.operation.validate.type;
+
+/**
+ * The Mode Type indicates a valid resource validation mode code.
+ * @see <a href="http://hl7.org/fhir/resource-validation-mode">resource validation mode</a>
```

Comment:
Somewhat minor, but because we implement a specific version of the FHIR specification, I generally use the version-specific URLs when linking to the spec from our code.
```suggestion
 * @see <a href="https://hl7.org/fhir/R4B/codesystem-resource-validation-mode.html">resource validation mode</a>
```

Reply:
This has been fixed by using the version-specific URL https://hl7.org/fhir/R4B/codesystem-resource-validation-mode.html